ocs_ci/ocs/mcg_workload.py

```diff
@@ -222,3 +222,20 @@ def _retrieve_job_state():
             log.debug(f"Logs from job pod {job_pod['metadata']['name']}: {pod_logs}")
 
         return False
+
+
+def get_pod_logs(job):
+    """
+    Get logs from job pod
+    Args:
+        job (obj): OCS job object
+
+    """
+    job_name = job.name
+    job_pods = pod.get_pods_having_label(f"job-name={job_name}", job.namespace)
+    for job_pod in job_pods:
+        log.info(f"Description of job pod {job_pod['metadata']['name']}: {job_pod}")
+        pod_logs = pod.get_pod_logs(
+            job_pod["metadata"]["name"], namespace=job_pod["metadata"]["namespace"]
+        )
+        log.info(f"Logs from job pod {job_pod['metadata']['name']}: {pod_logs}")
```

Comment:
I suggest to add a return statement to this function so that logs can be further processed. Otherwise, please change name of this function to 'print_pod_logs`.

Reply:
acked.